rules/windows/privilege_escalation_dns_serverlevelplugindll.toml

```diff
@@ -0,0 +1,55 @@
+[metadata]
+creation_date = "2024/05/29"
+integration = ["endpoint"]
+maturity = "production"
+updated_date = "2024/05/29"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies unusual DLLs loaded by the DNS Server process, potentially indicating the abuse of the ServerLevelPluginDll
+functionality. This can lead to privilege escalation and remote code execution with SYSTEM privileges.
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.library-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential DNS Server Privilege Escalation via ServerLevelPluginDll"
```

Comment:
rule name can be a bit adjusted to `Unsigned DLL loaded by DNS Service` to make it more adequate with the query, the ServerLevelPluginDll is already in the rule description and could be one of the reasons an unsigned DLL is loaded by DNS (there are other cases like AV DLLs or other custom SW).

Reply:
https://github.com/elastic/endpoint-rules/pull/3540